2023-02-23 14:56:03,986 INFO    MainThread:9120 [wandb_setup.py:_flush():76] Configure stats pid to 9120
2023-02-23 14:56:03,986 INFO    MainThread:9120 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Thomas\.config\wandb\settings
2023-02-23 14:56:03,986 INFO    MainThread:9120 [wandb_setup.py:_flush():76] Loading settings from c:\Users\Thomas\AudioHandling\wandb\settings
2023-02-23 14:56:03,987 INFO    MainThread:9120 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-02-23 14:56:03,987 INFO    MainThread:9120 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-23 14:56:03,987 INFO    MainThread:9120 [wandb_init.py:_log_setup():461] Logging user logs to c:\Users\Thomas\AudioHandling\wandb\run-20230223_145603-1t4ty0hk\logs\debug.log
2023-02-23 14:56:03,990 INFO    MainThread:9120 [wandb_init.py:_log_setup():462] Logging internal logs to c:\Users\Thomas\AudioHandling\wandb\run-20230223_145603-1t4ty0hk\logs\debug-internal.log
2023-02-23 14:56:03,990 INFO    MainThread:9120 [wandb_init.py:_jupyter_setup():411] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000158237EBD60>
2023-02-23 14:56:03,990 INFO    MainThread:9120 [wandb_init.py:init():495] calling init triggers
2023-02-23 14:56:03,990 INFO    MainThread:9120 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2023-02-23 14:56:03,990 INFO    MainThread:9120 [wandb_init.py:init():514] re-initializing run, found existing run on stack: 116l5eqn
2023-02-23 14:56:03,991 INFO    MainThread:9120 [wandb_run.py:_finish():1680] finishing run tnapier/a2o-swav-lightly/116l5eqn
2023-02-23 14:56:03,991 INFO    MainThread:9120 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2023-02-23 14:56:03,992 INFO    MainThread:9120 [wandb_run.py:_restore():1834] restore
2023-02-23 14:56:04,905 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 759
}

2023-02-23 14:56:05,277 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 759
}

2023-02-23 14:56:05,841 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11950
}

2023-02-23 14:56:05,958 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11950
}

2023-02-23 14:56:06,066 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11950
}

2023-02-23 14:56:06,175 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11950
}

2023-02-23 14:56:06,283 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11950
}

2023-02-23 14:56:06,391 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:06,500 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:06,608 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:06,717 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:06,825 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:06,934 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:07,042 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:07,150 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:07,259 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:07,367 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:07,611 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}

2023-02-23 14:56:08,338 INFO    MainThread:9120 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11950
  total_bytes: 11950
}
local_info {
}

2023-02-23 14:56:09,456 INFO    MainThread:9120 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2023-02-23 14:56:09,456 INFO    MainThread:9120 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2023-02-23 14:56:09,458 INFO    MainThread:9120 [wandb_run.py:_footer_sync_info():3055] logging synced files
2023-02-23 14:56:09,460 INFO    MainThread:9120 [wandb_init.py:init():548] starting backend
2023-02-23 14:56:09,460 INFO    MainThread:9120 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2023-02-23 14:56:09,463 INFO    MainThread:9120 [backend.py:ensure_launched():217] starting backend process...
2023-02-23 14:56:09,499 INFO    MainThread:9120 [backend.py:ensure_launched():222] started backend process with pid: 33640
2023-02-23 14:56:09,499 INFO    MainThread:9120 [wandb_init.py:init():558] backend started and connected
2023-02-23 14:56:09,501 INFO    MainThread:9120 [wandb_run.py:_label_probe_notebook():1084] probe notebook
2023-02-23 14:56:09,501 INFO    MainThread:9120 [wandb_run.py:_label_probe_notebook():1094] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-23 14:56:09,502 INFO    MainThread:9120 [wandb_init.py:init():634] updated telemetry
2023-02-23 14:56:09,523 INFO    MainThread:9120 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2023-02-23 14:56:39,528 INFO    MainThread:9120 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2023-02-23 14:56:39,528 ERROR   MainThread:9120 [wandb_init.py:init():674] backend process timed out
2023-02-23 14:56:39,528 ERROR   MainThread:9120 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2023-02-23 14:56:39,533 ERROR   MainThread:9120 [wandb_init.py:init():1070] error
Traceback (most recent call last):
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\wandb_init.py", line 1043, in init
    wandb will automatically resume the run with the id. Otherwise,
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\wandb_init.py", line 689, in init
    run._set_library(self._wl)
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\backend\backend.py", line 246, in cleanup
    """Connect to server."""
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 475, in join
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\interface\interface.py", line 666, in join
    self._publish_pause(pause)
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 472, in _communicate_shutdown
    return None
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 226, in _communicate
    metric: Optional[pb.MetricRecord] = None,
  File "c:\Users\Thomas\conda\envs\audiohandling\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 231, in _communicate_async
    preempting: Optional[pb.RunPreemptingRecord] = None,
Exception: The wandb backend process has shutdown
